### ISSUE TITLE
CORS issue while running in browser after installing whitelist.

### ISSUE THREAD

**Issue by gurindernarang** [drrasic/ionic_twitter_feed#5]
![cors twitter](https://cloud.githubusercontent.com/assets/10511236/17128979/e7920e5a-532d-11e6-8d71-13e342af56ec.png)
If i run the same API on a REST CLIENT, it returns a proper response.

**Comment by chsxe** [comment]
Can you be more specific with your problem? 
Perhaps this POST is the problem...
/oauth2/token HTTP/1.1

**Comment by gurindernarang** [comment]
After installing the whitelist pugin and adding meta tag in index.html
<meta http-equiv="Content-Security-Policy"
content="default-src *; style-src 'self' data: gap: 'unsafe-inline';
script-src 'self' 'unsafe-inline' 'unsafe-eval'">
still there is CORS issue in the application.
i m using following versions :
ionic version _1.7.15_
cordova version _6.2.0_
npm version _3.5.2_

**Comment by chsxe** [comment]
I am not too sure, but I checked it against the code and you have "data: gap" in the meta tag, is this intentional?

**Comment by gurindernarang** [comment]
yes, data: is used to allow loading resources via data scheme and gap: is
required in iOS(when using UIWebView).

**Comment by drrasic** [comment]
If running it browser have you tried spoofing the CORS header with something like this plugin for Chrome.
[CORS](https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?hl=en-US)

Or just build it on a device. It still works this way for me.
